src/subscription_manager/repofile.py

```diff
@@ -461,19 +461,28 @@ def sections(self):
 
         def fix_content(self, content):
             # Luckily apt ignores all Fields it does not recognize
-            baseurl = content["baseurl"]
+            parsed_url = urlparse(unquote(content["baseurl"]))
+            baseurl = parsed_url._replace(query="").geturl()
             url_res = re.match(r"^https?://(?P<location>.*)$", baseurl)
             ent_res = re.match(r"^/etc/pki/entitlement/(?P<entitlement>.*).pem$", content["sslclientcert"])
             if url_res and ent_res:
                 location = url_res.group("location")
                 entitlement = ent_res.group("entitlement")
                 baseurl = "katello://{}@{}".format(entitlement, location)
 
+            query = parse_qs(parsed_url.query)
+            if "rel" in query and "comp" in query:
+                suites = query["rel"][0].replace(",", " ")
+                components = query["comp"][0].replace(",", " ")
+            else:
+                suites = "default"
+                components = "all"
```

Comment:
are both `rel` and `comp` query arguments supposed to always be together? can it happen that only one (either of them) be part of the URL? I started to read the katello PR to try to understand that, but sadly I don't ruby...